src/components/LogView/LogLineText.js

```diff
@@ -16,11 +16,51 @@ type Props = {
   port: ?string,
   text: string,
   highlightText: string[],
-  handleDoubleClick: ()=> void,
+  handleDoubleClick: () => void,
+  prettyPrint: boolean,
+}
+
+export function findJSONObjectsInLine(text: string) {
```

Comment:
This isn't typescript, but this function should have a return type (actually I'm kind of surprised flow isn't complaining about this)

Reply:
What do you mean? Isn't the return type here void? I didn't add in this handler prop so I'm not sure how it should be different.

Reply:
Nope, you return a value at the bottom here. The return type is being inferred which is why flow didn't complain, but it would be helpful to have it declared explicitly